src/test/java/com/snowflake/kafka/connector/dlq/InMemoryKafkaRecordErrorReporter.java

```diff
@@ -0,0 +1,36 @@
+package com.snowflake.kafka.connector.dlq;
+
+import static java.util.Collections.unmodifiableList;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.kafka.connect.sink.SinkRecord;
+
+/* In memory implementation of KafkaRecordErrorReporter which mimics sending records to DLQ. Here we simply insert records into an ArrayList*/
```

Comment:
This is only used for testing?

Reply:
Yes, because there is no way right now to test with IT and see the contents in DLQ. (Also, it is in src/test folder)

Reply:
I did check manually though by inserting bad records that contents were delivering to DLQ topic.

Reply:
sounds good, could you add it to the comment?

Reply:
Will do!